src/main/webapp/app/modeling-submission/modeling-submission.component.ts

```diff
@@ -494,4 +466,11 @@ export class ModelingSubmissionComponent implements OnInit, OnDestroy, Component
         this.showComplaintForm = false;
         this.showRequestMoreFeedbackForm = !this.showRequestMoreFeedbackForm;
     }
+
+    /**
+     * The exercise is still active if it's due date hasn't passed yet, or if the participation was started after the due date has already passed
+     */
+    get isActive() {
+        return this.modelingExercise && (!this.modelingExercise.dueDate || moment(this.modelingExercise.dueDate).isAfter(moment()));
```

Comment:
Yes but I do it with isLate property. So if the exercise was started after the due date, student will see `late submission` button

Reply:
From our Slack discussion: 
> except the participation start date is after the exercise due date, then they can submit forever

Also, please change the comment then, because there you say that `isActive` checks if the participation was started after the due date. It's fine if you use the `isLate` property, but then the JSDoc should reflect that.

Reply:
@ungaralex True, I forgot to update JSDoc after the latest changes.